tests/Unit/Evolution/Systems/ScalarAdvection/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY Test_ScalarAdvection)
+
+set(LIBRARY_SOURCES
+  Test_Tags.cpp
+  )
+
+add_test_library(
+  ${LIBRARY}
+  "Evolution/Systems/ScalarAdvection"
+  "${LIBRARY_SOURCES}"
+  ""
+  )
+
+target_link_libraries(
+  ${LIBRARY}
+  PUBLIC
+  ScalarAdvection
+  DataStructures
+  Time
```

Comment:
Delete `Time` for now. Not sure if it's needed at all later when you add more tests.